### PR TITLE
permissions: document/holding/item resource policies refactoring

### PR DESCRIPTION
**Pull request by zannkukai**
Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

### CODE REVIEW

**Review by lauren-d — APPROVED** [review]

File: data/role_policies.json [hunk 1]
```diff
@@ -1,4 +1,60 @@
...
+    "document_importer",
+    "pro_full_permissions",
+    "pro_catalog_manager",
+    "pro_library_administrator"
+  ],
+  "hold-create": [
+    "pro_full_permissions",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline data/role_policies.json:24]: Not necessary if you have the role of a `pro_catalog_manager`

File: data/role_policies.json [hunk 2]
```diff
@@ -1,4 +1,60 @@
...
+  ],
+  "hold-update": [
+    "pro_full_permissions",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline data/role_policies.json:30]: Not necessary if you have the role of a `pro_catalog_manager`

File: data/role_policies.json [hunk 3]
```diff
@@ -1,4 +1,60 @@
...
+  ],
+  "hold-delete": [
+    "pro_full_permissions",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline data/role_policies.json:36]: Not necessary if you have the role of a `pro_catalog_manager`

File: data/role_policies.json [hunk 4]
```diff
@@ -1,4 +1,60 @@
...
+  ],
+  "item-create": [
+    "pro_full_permissions",
+    "pro_circulation_manager",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline data/role_policies.json:43]: Not necessary if you have the role of a `pro_catalog_manager`

File: data/role_policies.json [hunk 5]
```diff
@@ -1,4 +1,60 @@
...
+  ],
+  "item-update": [
+    "pro_full_permissions",
+    "pro_circulation_manager",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline data/role_policies.json:50]: Not necessary if you have the role of a `pro_catalog_manager`

File: data/role_policies.json [hunk 6]
```diff
@@ -1,4 +1,60 @@
...
+  ],
+  "item-delete": [
+    "pro_full_permissions",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline data/role_policies.json:56]: Not necessary if you have the role of a `pro_catalog_manager`

File: rero_ils/modules/holdings/permissions.py [hunk 7]
```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 #
 # RERO ILS
-# Copyright (C) 2020 RERO
-# Copyright (C) 2020 UCLouvain
+# Copyright (C) 2022 RERO
+# Copyright (C) 2022 UCLouvain
```
Comment (Garfield-fr) [inline rero_ils/modules/holdings/permissions.py:5]: maybe
```
# Copyright (C) 2020-2022 RERO
# Copyright (C) 2020-2022 UCLouvain
```

File: tests/data/policies/role_policies.json [hunk 8]
```diff
@@ -1,4 +1,57 @@
...
+  "doc-delete": [
+    "pro_full_permissions",
+    "pro_catalog_manager",
+    "pro_library_administrator"
+  ],
+  "hold-create": [
+    "pro_full_permissions",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline tests/data/policies/role_policies.json:21]: Not necessary if you have the role of a `pro_catalog_manager`

File: tests/data/policies/role_policies.json [hunk 9]
```diff
@@ -1,4 +1,57 @@
...
+  ],
+  "hold-update": [
+    "pro_full_permissions",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline tests/data/policies/role_policies.json:27]: Not necessary if you have the role of a `pro_catalog_manager`

File: tests/data/policies/role_policies.json [hunk 10]
```diff
@@ -1,4 +1,57 @@
...
+  ],
+  "hold-delete": [
+    "pro_full_permissions",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline tests/data/policies/role_policies.json:33]: Not necessary if you have the role of a `pro_catalog_manager`

File: tests/data/policies/role_policies.json [hunk 11]
```diff
@@ -1,4 +1,57 @@
...
+  ],
+  "item-create": [
+    "pro_full_permissions",
+    "pro_circulation_manager",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline tests/data/policies/role_policies.json:40]: Not necessary if you have the role of a `pro_catalog_manager`

File: tests/data/policies/role_policies.json [hunk 12]
```diff
@@ -1,4 +1,57 @@
...
+  ],
+  "item-update": [
+    "pro_full_permissions",
+    "pro_circulation_manager",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline tests/data/policies/role_policies.json:47]: Not necessary if you have the role of a `pro_catalog_manager`

File: tests/data/policies/role_policies.json [hunk 13]
```diff
@@ -1,4 +1,57 @@
...
+  ],
+  "item-delete": [
+    "pro_full_permissions",
+    "pro_catalog_manager",
+    "pro_library_administrator",
+    "pro_acquisition_manager"
```
Comment (Garfield-fr) [inline tests/data/policies/role_policies.json:53]: Not necessary if you have the role of a `pro_catalog_manager`

**Review by Garfield-fr — APPROVED** [review]

**Review by benerken — APPROVED** [review]

**Pull request merged**